cmd/auth/login.go

```diff
@@ -245,3 +253,33 @@ func getProfileName(authArguments *auth.AuthArguments) string {
 	split := strings.Split(host, ".")
 	return split[0]
 }
+
+func getClusterID(ctx context.Context, profileName string) (string, error) {
```

Comment:
Instead of directly reading from the config file, can you modify the LoadProfiles method directly for the DefaultProfiler?

You could add ClusterId to the Profile struct as well to make that work:
```
type Profile struct {
	Name      string
	Host      string
	AccountID string
}
```

That way, this function can simply call LoadProfiles.